repose-aggregator/artifacts/extensions-filter-bundle/src/main/application/WEB-INF/web-fragment.xml

```diff
@@ -32,4 +32,8 @@
         <filter-name>saml-policy</filter-name>
         <filter-class>org.openrepose.filters.samlpolicy.SamlPolicyTranslationFilter</filter-class>
     </filter>
+    <filter>
+        <filter-name>attribute-mapping-policy-validation</filter-name>
```

Comment:
I like it. It deters users from using it that shouldn't and the one that should only has to write it once.